src/extra/Markdown/index.tsx

```diff
@@ -208,6 +208,17 @@ const decoratorPlugin = (options: DecoratorPluginOptions) => {
 						},
 					);
 				} else {
+					children.push({
+						type: 'element',
+						tagName: 'span',
+						children: [
+							{
+								type: 'text',
+								value: text.substring(lastIndex),
+							},
+						],
+					});
+
```

Comment:
what does this fix?

Reply:
There can be a text left after the last match, like this:

```
"start @thundron end"
```

This code adds the `end` part.